### ISSUE TITLE
griffin-lim implementation for quick sanity check of tacotron output

### ISSUE THREAD

**Issue by zhouyong64** [dathudeptrai/TensorflowTTS#62]
It'd be helpful if there is some griffin-lim code to check if tacotron training is OK before training vocoders like MelGAN.

**Comment by dathudeptrai** [comment]
@zhouyong64 i think a plot of mel-spectrogram, the loss value and the alignment is enough to check the quality of tacotron output. But, add GL is a good idea, i will add it asap :D.

**Comment by zhouyong64** [comment]
> @zhouyong64 i think a plot of mel-spectrogram, the loss value and the alignment is enough to check the quality of tacotron output. But, add GL is a good idea, i will add it asap :D.

Thanks. I understand your point. If someone modified lots of things based on your code, having a quick way to syntheisize audio might be the best indicator as to whether the modification works.

**Comment by dathudeptrai** [comment]
@zhouyong64 i add GL-Tensorflow implementation here (https://github.com/dathudeptrai/TensorflowTTS/blob/master/tensorflow_tts/utils/griffin_lim.py). I also sanity check the output of GL on notebook (https://github.com/dathudeptrai/TensorflowTTS/blob/master/notebooks/Griffin_lim_tensorflow.ipynb)
